generators/add-action/asset-compute/index.js

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+// #########################################################################################
+// 3rd party template
+// #########################################################################################
+
+const path = require('path')
+const ActionGenerator = require('../../../lib/ActionGenerator')
+
+class AssetComputeGenerator extends ActionGenerator {
+  constructor (args, opts) {
+    super(args, opts)
+    this.props = {}
+  }
+
+  async prompting () {
+    this.props.actionName = await this.promptForActionName('example of a custom worker for the Adobe Asset Compute service', 'worker-example')
```

Comment:
What about naming it `worker` instead of `worker-example`?

This is asked during `aio app init` and used as default:

```
? We are about to create a new sample action that example of a custom worker for the Adobe Asset Compute service
how would you like to name this action? worker-example
```

If folks hit just enter, all their projects will have `worker-example`, which seems weird.

Reply:
+1 for just `worker`

Reply:
(Name changed, unit tests updated accordingly).